src/services/import.service.js

```diff
@@ -9,21 +9,19 @@ const logger = require('../server/logger');
  * @param {Object} res The response object returned to the client by the server
  */
 async function bulkImport(req, res) {
-  logger.info('Base >>> $import');
+  logger.info('Base >>> Import');
   logger.debug(`Request headers: ${JSON.stringify(req.header)}`);
   logger.debug(`Request body: ${JSON.stringify(req.body)}`);
   logger.debug(`Request params: ${JSON.stringify(req.params)}`);
 
   // ID assigned to the requesting client
-  const clientEntry = await addPendingBulkImportRequest();
+  const clientEntry = await addPendingBulkImportRequest(req.body);
 
-  const exportURL = retrieveExportUrl(req.body.parameter);
-  const exportType = retrieveExportType(req.body.parameter);
+  const inputUrls = retrieveInputUrls(req.body.parameter);
```

Comment:
This adds the bulk import request to the database before it has been validated to be a good request. Perhaps this should happen later after its been validated to be a good request.

Reply:
I think the code I changed in this file may make more sense and make this OBE but lmk!

Reply:
Yep. This makes more sense now that the error is recorded for return on the bulk status endpoint.